static/index.css

```diff
@@ -0,0 +1,62 @@
+* {
+    margin: none;
+    padding: none;
+    box-sizing: border-box;
+}
+
+body{
+    background-color: rgb(177, 177, 177);
+}
+h1{
+    color: lightslategrey;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 30px;
+}
+
+.header-flex {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+
+.main-flex {
+    
+    
+}
+
+.crit-timeline {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+    height: 25%;
+    margin: 20px 20px;
+    color: rgb(3, 20, 26);
+    border: 1px solid rgb(199, 196, 176);
+    background-color: rgb(225, 223, 223);
+    border-radius: 8px;
+    box-shadow: lightslategrey 5px 5px;
+    align-items: flex-start;
+}
+
+.crit-timeline p {
+    margin: 20px;
+    
+}
+.crit-timeline h3 {
+    margin-left: 15px;
+}
+
+button {
+   
```

Comment:
Let's tidy all of this up in the future before submitting PRs. Clear out any un-used whitespace, remove any selectors that are empty (like button{} above)